providers/ringcentral/src/lib/ringcentral.provider.ts

```diff
@@ -0,0 +1,67 @@
+import {
+  ChannelTypeEnum,
+  ISendMessageSuccessResponse,
+  ISmsOptions,
+  ISmsProvider,
+} from '@novu/stateless';
+
+const RC = require('@ringcentral/sdk').SDK;
+require('dotenv').config();
+
+export class RingcentralSmsProvider implements ISmsProvider {
+  channelType = ChannelTypeEnum.SMS as ChannelTypeEnum.SMS;
+  platform = rcsdk.platform();
+  RECIPIENT = process.env.SMS_RECIPIENT;
```

Comment:
The recipient should come from the ISmsOptions in the `sendMessage` method so we can send dynamically :)